src/cljs/proton/layers/core/keybindings.cljs

```diff
@@ -162,6 +162,9 @@
              :n {:title "line numbers"
                  :target (fn [atom] (.getView (.-views atom) (.getActiveTextEditor (.-workspace atom))))
                  :action "editor:toggle-line-numbers"}
+             :h {:category "highlight"
+                 :h {:title "highlight current line"
```

Comment:
All right - let's leave it on highlights then. We can always change it later. 
Though you could change the keybinding to `h l`? It's highlighting the current line and not highlighting the highlight :smile: